src/panels/SkyBrowserPanel/ImageList/ImageList.tsx

```diff
@@ -17,14 +19,18 @@ interface Props {
 // Memoizing this as it is very expensive
 export const ImageList = memo(function ImageList({ imageList, noImagesDisplay }: Props) {
   const { width } = useWindowSize();
+  const { allowedKeys, toggleKey, selectedKeys } = useKeySettings<SkyBrowserImage>({
+    collection: true,
+    name: true
```

Comment:
I would opt for always just including the name per default? 

Or at least, always put the name as the first key in the list:
```suggestion
    name: true,
    collection: true
```

Reply:
I ordered them alphabetically which makes sense to me. We can not assume that the objects will contain a key called name so we can't make that generalization